src/withdraw-utils.js

```diff
@@ -24,18 +24,56 @@ function hashInputsWithdraw(inputs){
 
     // inputs strings hexadecimal
     let strRootState = utils.padZeros(inputs.rootState.toString("16"), rootStateB / 4);
-    let strEthAddr = utils.padZeros(inputs.ethAddr.toString("16"), ethAddrB / 4);
+    let strEthAddrState = utils.padZeros(inputs.ethAddrState.toString("16"), ethAddrB / 4);
     let strTokenID = utils.padZeros(inputs.tokenID.toString("16"), tokenIDB / 4);
     let strExitBalance = utils.padZeros(inputs.exitBalance.toString("16"), exitBalanceB / 4);
     let strIdx = utils.padZeros(inputs.idx.toString("16"), idxB / 4);
 
     // build final inputs string
-    const finalStr = strRootState.concat(strEthAddr).concat(strTokenID).concat(strExitBalance)
+    const finalStr = strRootState.concat(strEthAddrState).concat(strTokenID).concat(strExitBalance)
         .concat(strIdx);
 
     return utils.sha256Snark(finalStr);
 }
 
+
+/**
+ * @param {Object} inputs - Object containing all withdraw Bjj inputs 
+ * @return {Scalar} hash global inputs with sha256 % rField 
+ */
+function hashInputsWithdrawBjj(inputs){
```

Comment:
Missing unit test for this function.
Please add test vectors similar to https://github.com/hermeznetwork/commonjs/blob/master/test/withdraw-utils.test.js#L8